File: mycollab-web/src/main/java/com/esofthead/mycollab/module/crm/view/opportunity/OpportunityListView.java
/**
 * This file is part of mycollab-web.
 *
 * mycollab-web is free software: you can redistribute it and/or modify
 * it under the terms of the GNU General Public License as published by
 * the Free Software Foundation, either version 3 of the License, or
 * (at your option) any later version.
 *
 * mycollab-web is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 * GNU General Public License for more details.
 *
 * You should have received a copy of the GNU General Public License
 * along with mycollab-web.  If not, see <http://www.gnu.org/licenses/>.
 */
package com.esofthead.mycollab.module.crm.view.opportunity;

import com.esofthead.mycollab.module.crm.domain.SimpleOpportunity;
import com.esofthead.mycollab.module.crm.domain.criteria.OpportunitySearchCriteria;
import com.esofthead.mycollab.vaadin.web.ui.ListView;

/**
 * @author MyCollab Ltd.
 * @since 1.0
 */
public interface OpportunityListView extends ListView<OpportunitySearchCriteria, SimpleOpportunity> {
    String VIEW_DEF_ID = "crm-opportunity-list";

}

File: mycollab-web/src/main/java/com/mycollab/module/crm/view/opportunity/OpportunityListView.kt
package com.mycollab.module.crm.view.opportunity

import com.mycollab.module.crm.domain.SimpleOpportunity
import com.mycollab.module.crm.domain.criteria.OpportunitySearchCriteria
import com.mycollab.vaadin.web.ui.IListView

/**
 * @author MyCollab Ltd.
 * @since 1.0
 */
interface OpportunityListView : IListView<OpportunitySearchCriteria, SimpleOpportunity>
